SparseMatrixTest.java

```diff
@@ -0,0 +1,100 @@
+package sparseMatrix;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SparseMatrixTest {
+	// test for null value as input
+	@Test
+	public void testMatrixNullArray() {
+		try {
+			new SparseMatrix(null);
+		} catch (Exception e) {

```

Comment:
If you know that this error is specific then handle the specific error, Why handling Generic error? Update the code.